### PR TITLE
Resolves #91

### LINKED ISSUE #91: Change format of .lib files from multiple lines to single lines
**Issue details-**
- Currently the generated `.lib` files follow the multi line format, change it to single line format for easier use
- Device models to be generated according to the Spice's convention. Example -
- Multi Line format -
```
.MODEL PowerDiode D(
+ Vj=.75
+ Nbvl=14.976
+ Cjo=175p
+ Rs=.25
+ Isr=1.859n
+ Eg=1.11
)
```
- Single Line format -
```
.MODEL PowerDiode D( Vj=.75 Nbvl=14.976 Cjo=175p Rs=.25 Isr=1.859n Eg=1.11 )
```
Edit: Add relevant tags such as `feature` or `enhancement`
Edit: Added spice format for reference

### PR DESCRIPTION
**Pull request by nilshah98**
**PR Details-**

**Resolves** #91
- Changed format of `.lib` files from multiple line format to single line format

### CODE REVIEW

File: src/modelEditor/ModelEditor.py [hunk 1]
```diff
@@ -492,10 +492,10 @@ def createXML(self, model_name):
                 self.modelname +
                 ' ' +
                 self.model_name +
-                '(\n')
+                '(')
             for tags, text in list(self.modeldict.items()):
-                txtfile.write('+ ' + tags + '=' + text + '\n')
-            txtfile.write(')')
+                txtfile.write(' ' + tags + '=' + text)
+            txtfile.write(' )')
```
Comment (sunilshetye) [inline src/modelEditor/ModelEditor.py]:
```suggestion
            txtfile.write(' )\n')
```

File: src/modelEditor/ModelEditor.py [hunk 2]
```diff
@@ -513,10 +513,10 @@ def createXML(self, model_name):
...
```
Comment (sunilshetye) [inline src/modelEditor/ModelEditor.py]:
```suggestion
            txtfile.write(' )\n')
```

File: src/modelEditor/ModelEditor.py [hunk 3]
```diff
@@ -534,10 +534,10 @@ def createXML(self, model_name):
...
```
Comment (sunilshetye) [inline src/modelEditor/ModelEditor.py]:
```suggestion
            txtfile.write(' )\n')
```

File: src/modelEditor/ModelEditor.py [hunk 4]
```diff
@@ -555,10 +555,10 @@ def createXML(self, model_name):
...
```
Comment (sunilshetye) [inline src/modelEditor/ModelEditor.py]:
```suggestion
            txtfile.write(' )\n')
```

File: src/modelEditor/ModelEditor.py [hunk 5]
```diff
@@ -576,10 +576,10 @@ def createXML(self, model_name):
...
```
Comment (sunilshetye) [inline src/modelEditor/ModelEditor.py]:
```suggestion
            txtfile.write(' )\n')
```

File: src/modelEditor/ModelEditor.py [hunk 6]
```diff
@@ -597,10 +597,10 @@ def createXML(self, model_name):
...
```
Comment (sunilshetye) [inline src/modelEditor/ModelEditor.py]:
```suggestion
            txtfile.write(' )\n')
```

File: src/modelEditor/ModelEditor.py [hunk 7]
```diff
@@ -645,10 +645,10 @@ def savethefile(self, editfile):
             self.ref_model +
             ' ' +
             self.model_name +
-            '(\n')
+            '(')
         for tags, text in list(self.modeldict.items()):
-            libfile.write('+  ' + tags + '=' + text + '\n')
-        libfile.write(')')
+            libfile.write(' ' + tags + '=' + text)
+        libfile.write(' )')
```
Comment (sunilshetye) [inline src/modelEditor/ModelEditor.py]:
```suggestion
        libfile.write(' )\n')
```

**Comment by nilshah98** [comment]
The recommended changes have been made, please review @sunilshetye

**Pull request merged**